cpp/tests/error/error_handling_test.cu

```diff
@@ -50,6 +50,8 @@ CUDF_KERNEL void test_kernel(int* data) { data[threadIdx.x] = threadIdx.x; }
 // calls.
 TEST(StreamCheck, FailedKernel)
 {
+  if (getenv("LIBCUDF_MEMCHECK_ENABLED")) { return; }
```

Comment:
perhaps we should use `GTEST_SKIP()` rather than just returning?

```suggestion
  if (getenv("LIBCUDF_MEMCHECK_ENABLED")) { GTEST_SKIP(); }
```

Reply:
Ok. This works in this file but not in the `test_default_stream_identification.cu` since that one is not actually a gtest.